src/backend_tools/db_provider.ts

```diff
@@ -118,7 +123,7 @@ class DbProvider {
   private async getEventRefsFromClub(clubName: string) {
     throw new Error('Unfinished');
 
-    const club = await this.getClub(clubName);
+    const club = await this.getClubById(clubName);
```

Comment:
Is ID just name?

Reply:
That's just a remnant from the original implemetation, I'll change that whenever we are able to implement actual events